specification/playwrighttesting/PlaywrightTesting/tspconfig.yaml

```diff
@@ -14,3 +14,32 @@ options:
     examples-directory: "{project-root}/examples"
     output-file: "{azure-resource-provider-folder}/{service-name}/{version-status}/{version}/playwrighttesting.json"
     omit-unreachable-types: true
+  "@azure-tools/typespec-python":
+    package-dir: "azure-developer-microsoftplaywrighttesting"
+    package-name: "{package-dir}"
+    # emitter-output-dir: "{python-sdk-folder}/sdk/{service-directory-name}/{package-name}"
+    package-mode: azure-dataplane
+    flavor: azure
+  "@azure-tools/typespec-ts":
+    package-dir: "microsoft-playwright-testing-rest"
```

Comment:
I am not sure whether js should have `microsoft` in package name @MaryGao

Reply:
Usually `no`. We don't prefix `microsoft` or `azure` in package dir.

Reply:
Agree that SDK package name shall not contain "azure" or "microsoft"

Reply:
Address in another comment